api/data/page-filters/page-filters-csv-parser.ts

```diff
@@ -0,0 +1,91 @@
+import * as fs from 'fs';
+import { Options, parse } from 'csv-parse';
+
+const GROUP_HEADING_REGEX = /^\(.*\)$/;
+
+const isGroupHeadingRow = (row: string[]): boolean => {
+  for (let idx = 0; idx < row.length; idx++) {
+    const column = row[idx];
+    if (GROUP_HEADING_REGEX.test(column) === true) {
+      return true;
+    }
```

Comment:
Small thing but you can just check for `if (GROUP_HEADING_REGEX.test(column))` since it's either true or false.

Reply:
True, I'm just used to writing code with strict checking.